js/ui/map.js

```diff
@@ -287,6 +287,27 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Set constraint on the map's geographical bounds.
+     *
+     * @param {LngLatBounds | Array<Array<number>>} lnglatbounds optional Desired max bounds of the map. If ommitted, function removes any bounds constraints on the map.
```

Comment:
FWIW, we use `null` to unset paint and layout properties https://github.com/mapbox/mapbox-gl-js/blob/bench-interface/js/style/style_layer.js#L116-L117